api/v1/checluster_types.go

```diff
@@ -360,6 +361,12 @@ type CheClusterSpecServer struct {
 	WorkspacePodNodeSelector map[string]string `json:"workspacePodNodeSelector,omitempty"`
 	// The pod tolerations put on the workspace pods to limit where the workspace pods can run.
 	WorkspacePodTolerations []corev1.Toleration `json:"workspacePodTolerations,omitempty"`
+	// The default editor to workspace create with.
+	// +optional
+	WorkspaceDefaultEditor string `json:"workspaceDefaultEditor,omitempty"`
+	// The default components to workspace create with.
+	// +optional
+	WorkspaceDefaultComponents []devfile.BaseComponent `json:"workspaceDefaultComponents,omitempty"`
```

Comment:
Are you sure about `devfile.BaseComponent`?  maybe `devfile.Component` ?

Reply:
I am not sure about BaseComponent or Component
@ibuziuk @amisevsk WDYT?

Reply:
should be Component https://github.com/eclipse/che/issues/21353#issuecomment-1136264410

Reply:
Yup, it should be Component.

Reply:
I have fixed it